v3/plugins/redhat/mta/0.0.46/meta.yaml

```diff
@@ -23,7 +23,7 @@ spec:
     attributes:
       protocol: http
   containers:
-  - image: "quay.io/windupeng/mta-vscode-extension:latest"
+  - image: "docker.io/windup3/rhamt-vscode-extension:java8"
```

Comment:
why undo this change? @johnsteele specifically told me to use this replacement image since HE put it there. Upstream for the win, right?